examples/pagemirror_extension/content_script.js

```diff
@@ -17,7 +17,6 @@ var observer;
 if (typeof WebKitMutationObserver != 'function') {
   console.error('This example extension requires MutationObservers. ' +
                 'Try the Chrome Canary build.');
-  return;
```

Comment:
Instead of this, you should use `MutationObserver` instead of `WebKitMutationObserver` all the way through.